src/main/java/edu/kit/hci/soli/service/UserService.java

```diff
@@ -28,11 +33,29 @@ public  UserService(UserRepository userRepository) {
         return userRepository.save(user);
     }
 
+    public void disableOrRenableUser(User user) {
+
+        if(!user.isDisabled()) {
+            bookingsService.deleteAllBookingsForUser(user);
+        }
+
+        user.setDisabled(!user.isDisabled());
+        userRepository.save(user);
+    }
+
+    public @NotNull List<User> getAllWithoutAdmin() {
+        return userRepository.findAllWithoutAdmin();
+    }
```

Comment:
This should have a name based on the domain. Maybe `getManageableUsers` or something?